.demo/env/clouds/aws/demo/jobs.demo.yaml

```diff
@@ -0,0 +1,44 @@
+teamConfig:
+    teams:
+        demo:
+            jobs:
+                - image:
+                      repository: busybox
+                      tag: latest
+                      pullPolicy: IfNotPresent
+                  name: regular-job
+                  runPolicy: Always
+                  script: echo "Hello World"
+                  type: job
+                - env:
+                      this-is: an env
+                  image:
+                      repository: busybox
+                      tag: latest
+                      pullPolicy: IfNotPresent
+                  name: job-with-env-and-secret
+                  runPolicy: OnSpecChange
+                  secret:
+                      this-is: a secret
+                  script: echo "Hello World again"
+                  type: job
+                - image:
+                      repository: busybox
+                      tag: latest
+                      pullPolicy: IfNotPresent
+                  name: regular-cronjob
+                  schedule: '0 1 * * *'
+                  script: echo "Hello World every day"
+                  type: cronjob
+                - env:
```

Comment:
env is fine, as we will still allow users to add env vars directly.

_But_ we will also let them create configmaps, just like secrets, so we will create `teams/config.*.yaml` which will contain configuration to deploy configmaps. Then these can be referenced by name as well (just like secrets), and array of keys selected.

Reply:
For both secrets and configmaps, for each key an optional path and subpath can be given: to allow users to give a path in the container where the key will be mounted (see example snippet I gave for schema before).

Reply:
Ok sounds good.